services/frontend/.env.production

```diff
@@ -1,8 +1,8 @@
-REACT_APP_GRAPHQL_HTTP_URL=https://graphql.eosrate.io/v1/graphql
-REACT_APP_API_URL=https://bp.cryptolions.io
-REACT_APP_GRAPHQL_WS_URL=wss://graphql.eosrate.io/v1/graphql
-REACT_APP_EOS_API_URL=https://bp.cryptolions.io
-REACT_APP_EOS_API_HOST=bp.cryptolions.io
+REACT_APP_GRAPHQL_HTTP_URL=https://graphql-jungle.eosrate.io/v1alpha1/graphql
+REACT_APP_API_URL=https://jungle.eosio.cr
+REACT_APP_GRAPHQL_WS_URL=wss://graphql-jungle.eosrate.io/v1alpha1/graphql
+REACT_APP_EOS_API_URL=https://jungle.eosio.cr
+REACT_APP_EOS_API_HOST=jungle.eosio.cr
 REACT_APP_EOS_API_PORT=443
 REACT_APP_EOS_API_PROTOCOL=https
-REACT_APP_EOS_CHAIN_ID=aca376f206b8fc25a6ed44dbdc66547c36c6c33e3a119ffbeaef943642f0e906
+REACT_APP_EOS_CHAIN_ID=2a02a0053e5a8cf73a56ba0fda11e4d92e0238a4a2aa74fccf46d5a910746840
```

Comment:
this seems like jungle chain ID, shouldnt it be the mainnet chain ID?

Reply:
@xavier506 I think it was needed to set the jungle chain values. Let me setup with the proper values to point to the mainnet